releases/2022.2/2022.2.md

```diff
@@ -0,0 +1,177 @@
+
+<div align="center">
+  <a href="assets/elasticsearch_kibana.png"><img src="assets/elasticsearch_kibana.png"></a>
+</div>
+<div align="center">
+  Figure 2 - Elasticsearch, Kibana, Filebeat and APM integration with Kytos.
+</div>
+
+### 6. Link liveness detection
+
+`of_lldp` has been extended to be able to provide [link liveness detection](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP030.rst), leveraging the existing LLDP messages, in order to detect link transport outage that might not necessarily trigger a port down event. Link liveness detection can be enabled per interface and it needs to be enabled on each pair in order to work. This implementation is meant to provide a generic alternative to BFD (Bidirectional Forwarding Detection - RFC 5880) where the protocol states is being tracked by the control plane.
+
+### 7. Interface loop detection
+
+`of_lldp` has also been augmented to detect if an interface of a switch is looped. [Interface loop detection](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP025.rst) is automatically enabled once LLDP since it is been designed to detect unexpected or accidental loops that can happen. This feature has configuration options to configure configure which action should be applied once a loop is detected, by default, it generates a log warning message, but it can also disable an interface. Also, loops can be configured to be ignored.
+
+### 8. New Web UI components
+
+The following Web UI functionalities were added on `topology`:
+
+- Table to display switch metadata.
+- Table to display links of a switch including its attributes.
+- Form and buttons to add and remove metadata from a switch, a link or an interface.
+- Toggle buttons to enable and disable a switch, a link or an interface.
+- Form and buttons to enable and disable LLDP on interfaces.
+
+<div align="center">
+  <a href="assets/topology_ui.png"><img src="assets/topology_ui.png"></a>
+</div>
+<div align="center">
+  Figure 3 - <code>topology</code> Web UI
+</div>
+
+These new Web UI features were added on `maintenance`:
+
+- Table to list, edit, and filter maintenance windows.
+- Toolbar to create maintenance windows.
+- Buttons to finish, extend or delete a maintenance window.
+
+<div align="center">
+  <a href="assets/maintenance_ui.png"><img src="assets/maintenance_ui.png"></a>
+</div>
+<div align="center">
+  Figure 4 - <code>maintenance</code> Web UI
+</div>
+
+### 9. Increase unit test coverage to 85% on `coloring`, `sdntrace` and `sdntrace_cp`
+
+`coloring`, `sdntrace`, and `sdntrace_cp` now have at least 85% of total unit test coverage, enhancing code quality and future code maintainability.
```

Comment:
here again, we can mention that all the Napps have coverage above 85%, and we can then mention those specific Napps. Wdyt?

Reply:
@italovalcy done here https://github.com/kytos-ng/kytos-ng.github.io/pull/34/commits/4980a9a785cd3fab3ec99da0dd3f1a7d0597acbb